Article/Article.js

```diff
@@ -2,15 +2,22 @@ const mongoose = require('mongoose');
 // const ObjectId = mongoose.Schema.Types.ObjectId;
 
 const Article = mongoose.Schema({
-    id: Number,
+    id: {
+        type: String,
+        unique: true
+    },
     name: String,
     url: String,
     timestamp: Date,
     description: String,
     keywords: [String],
     summary: String,
     content: String,
-    clickbait: Number
+    clickbait: String,
+    createdAt: {
+        type: Date,
+        default: Date.now()
```

Comment:
What is the createdAt property used for?

Reply:
the date and time the articles get entered into the database - our database.